internal/handler/common.go

```diff
@@ -0,0 +1,7 @@
+package handler
+
+import (
+	"github.com/SimplQ/simplQ-golang/internal/persistence"
+)
+
+var Store persistence.QueueStore
```

Comment:
common.go was a good idea. Let's do two things. 

1. Move `internal/handler/common.go` to `internal/datastore/store.go`.
2. Let's also initialise Store to the mongo client in this file.
3. Then import Store and use it in internal/handler/queue.go

Reply:
Rationale: Our `persistence` package would then expose the store to outside world, it's implementation and everything would stay inside it as private.

Reply:
Updated